data/reference/syntax.adoc

```diff
@@ -1,45 +1,58 @@
 = AsciiDoc Syntax
+:url-gem: http://rubygems.org/gems/asciidoctor
+:icons: font
 
 == Paragraphs
 
 A normal paragraph.
+Line breaks are not preserved.
 
-  An indented literal
-  paragraph.
+Use a blank line to separate paragraphs.
 
-== Formatted Text
+[%hardbreaks]
+This paragraph has the `hardbreaks` option.
+Notice that line breaks are now preserved.
 
-*bold text*.
-_emphasized text_.
-`monospaced text`.
+ A literal paragraph
+ disables text formatting
+ and preserves spaces and
+ line breaks.
 
-=== Unconstrained
+[sidebar#special.rolename]
+Add a style, ID, and/or role to give a paragraph special meaning, like this sidebar.
 
-In short, unconstrained means anywhere, including within a word:
+NOTE: An admonition paragraph, like this note, grabs the reader's attention.
 
-bold ** text with spaces **.
-emphasized le__tt__ers.
-m``o``n``o``spaced letters.
+== Text Formatting
 
-== Document Links
+*bold*
+_italic_ (aka emphasis)
+`monospace` (aka typewriter text)
 
-// define link target
-[[id]]
-// link to target id
-<<id,caption>>
-// link to an external file
-link:file#id[caption]
-// cross references between documents
-xref:sibling-document.adoc[caption]
+**C**reate+**R**ead+**U**pdate+**D**elete
+fan__freakin__tastic
+``mono``culture
 
-=== URLs
+== Attributes
 
-Use normal URL and email address syntax or:
+// reference built-in or custom attributes using {attribute-name}
+You can install Asciidoctor {asciidoctor-version} from {url-gem}.
 
-mailto:hello@domain.org[Say hello!]
-link:https://domain.org/page[Go to page]
-link:../path/to/file.txt[Read file]
-link:\\path\to\file.pdf[Download file.pdf]
+== Links
+
+https://example.org/page[A webpage]
+link:../path/to/file.txt[A local file]
+xref:document.adoc[A sibling document]
+mailto:hello@example.org[Email to say hello!]
+
+== Anchors
+
+[#idname]
+paragraph (or block) with an anchor (aka ID) 
+
+<<idname,optional link text>>footnote:[Link to anchor]
```

Comment:
I'm confused by the "footnote"

Reply:
I moved it to the anchors section as you suggested.

Reply:
I mean a separate paragraph in that section.